publication-commons/build.gradle

```diff
@@ -2,7 +2,22 @@ repositories {
     maven { url "https://s3.eu-central-1.amazonaws.com/dynamodb-local-frankfurt/release" }
 }
 
+def jacksonVersion = '2.10.2'
+def awsSdkVersion = '1.11.714'
+
 dependencies {
+    api group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: 'v0.3.0'
```

Comment:
Are you sure that implementation does not do the job?

Reply:
Yes, I didn't get access to transitive dependencies when using implementation